src/packages/@ncigdc/components/ActionsRow.js

```diff
@@ -72,46 +72,50 @@ export default compose(
           </Button>
 
           <DownloadManifestButton fileCount={totalFiles} filters={filters} style={{ margin: '5px 2px 0px 3px' }} />
-          {AWG || (filters
```

Comment:
Correcting this, as it was adding a superfluous "View XYZ cases in exploration".